components/audit_table/audit_table.tsx

```diff
@@ -630,8 +636,7 @@ export class AuditTable extends React.PureComponent {
         }
 
         const date = new Date(audit.create_at);
-        const auditInfo = {};
-        auditInfo.timestamp = (
+        const timestamp = (
```

Comment:
Why is this better than create an object and populate it? I'm asking this because if think is easier to understand if we use an object.

Reply:
Thanks for your review.
If we create an object and populate it,  we need to create an auditInfo object with default values.
 (The properties of AuditInfo type are non-null.)
Since these default values are meaningless, I did not create the object.

``` ts
const auditInfo: AuditInfo = {
    timestamp: <></>,
    userId: '',
    desc: '',
    ip: '',
    sessionId: '',
};
```